### PR TITLE
fix(.wordlist.txt): Update spellcheck wordlist

### PR DESCRIPTION
**Pull request by peterjunpark**
Greenlist new acronyms and vocabulary in `.wordlist.txt`

### CODE REVIEW

File: .wordlist.txt [hunk 1]
```diff
@@ -15,9 +15,12 @@ ASICs
 ASan
 ASm
 ATI
+AWQ
```
Comment (peterjunpark) [inline .wordlist.txt:18]: Activation-aware Weight Quantization

File: .wordlist.txt [hunk 2]
```diff
@@ -15,9 +15,12 @@ ASICs
...
+AdaLoRA
```
Comment (peterjunpark) [inline .wordlist.txt:19]: https://huggingface.co/docs/peft/en/package_reference/adalora

File: .wordlist.txt [hunk 3]
```diff
@@ -15,9 +15,12 @@ ASICs
...
 AddressSanitizer
 AlexNet
 Arb
+AutoGPTQ
```
Comment (peterjunpark) [inline .wordlist.txt:23]: https://github.com/AutoGPTQ/AutoGPTQ

File: .wordlist.txt [hunk 4]
```diff
@@ -56,6 +59,7 @@ CommonMark
 Concretized
 Conda
 ConnectX
+DDP
```
Comment (peterjunpark) [inline .wordlist.txt:62]: Distributed Data Parallel

File: .wordlist.txt [hunk 5]
```diff
@@ -86,6 +93,7 @@ FFmpeg
 FHS
 FMA
 FP
+FSDP
```
Comment (peterjunpark) [inline .wordlist.txt:96]: Fully-Sharded Data Parallel

File: .wordlist.txt [hunk 6]
```diff
@@ -381,10 +418,13 @@ api
 atmi
 atomics
 autogenerated
+autoregression
+autoregressive
```
Comment (peterjunpark) [inline .wordlist.txt:422]: https://en.wikipedia.org/wiki/Autoregressive_model

File: .wordlist.txt [hunk 7]
```diff
@@ -381,10 +418,13 @@ api
...
 avx
 awk
 backend
 backends
+backpropagation
```
Comment (peterjunpark) [inline .wordlist.txt:427]: https://en.wikipedia.org/wiki/Backpropagation

File: .wordlist.txt [hunk 8]
```diff
@@ -442,6 +485,7 @@ dimensionality
 disambiguates
 distro
 doxysphinx
+dropdown
```
Comment (peterjunpark) [inline .wordlist.txt:488]: Idk why this is flagged by the spellchecker. Pretty sure this is from an RST directive.

File: .wordlist.txt [hunk 9]
```diff
@@ -667,6 +722,7 @@ writebacks
 wrreq
 wzo
 xargs
+xFormers
```
Comment (peterjunpark) [inline .wordlist.txt:724]: https://github.com/facebookresearch/xformers

File: .wordlist.txt [hunk 10]
```diff
@@ -655,6 +709,7 @@ vectorized
 vectorizer
 vectorizes
 vjxb
+vLLM
```
Comment (peterjunpark) [inline .wordlist.txt]: https://docs.vllm.ai/en/latest/

File: .wordlist.txt [hunk 11]
```diff
@@ -313,19 +341,27 @@ TCIU
 TCP
 TCR
 TFLOPS
+TGI
+TPOT
 TPU
 TPUs
+TRL
+TTFT
```
Comment (peterjunpark) [inline .wordlist.txt:349]: time to first token

File: .wordlist.txt [hunk 12]
```diff
@@ -313,19 +341,27 @@ TCIU
 TCP
 TCR
 TFLOPS
+TGI
```
Comment (peterjunpark) [inline .wordlist.txt:344]: Hugging Face TGI

File: .wordlist.txt [hunk 13]
```diff
@@ -313,19 +341,27 @@ TCIU
...
+TPOT
```
Comment (peterjunpark) [inline .wordlist.txt:345]: time per output token

File: .wordlist.txt [hunk 14]
```diff
@@ -313,19 +341,27 @@ TCIU
...
 TPU
 TPUs
+TRL
```
Comment (peterjunpark) [inline .wordlist.txt:348]: technology readiness level

File: .wordlist.txt [hunk 15]
```diff
@@ -266,11 +287,14 @@ ROCmCC
...
 ROCmValidationSuite
 ROCr
+RPC
 RST
+RNN
 RW
 Radeon
 RelWithDebInfo
 Req
+ReLU
```
Comment (peterjunpark) [inline .wordlist.txt]: Rectified Linear Unit

File: .wordlist.txt [hunk 16]
```diff
@@ -266,11 +287,14 @@ ROCmCC
 ROCmSoftwarePlatform
 ROCmValidationSuite
 ROCr
+RPC
```
Comment (peterjunpark) [inline .wordlist.txt:291]: remote procedure call

File: .wordlist.txt [hunk 17]
```diff
@@ -165,21 +175,26 @@ LLMs
...
 LM
 LSAN
+LSTM
 LTS
+LinearReLU
 LoRA
 MEM
 MERCHANTABILITY
 MFMA
+MHA
```
Comment (peterjunpark) [inline .wordlist.txt:185]: Multi-Head Attention

File: .wordlist.txt [hunk 18]
```diff
@@ -165,21 +175,26 @@ LLMs
...
 MIOpenGEMM
 MIVisionX
 MLM
+MLP
 MMA
 MMIO
 MMIOH
 MNIST
 MPI
+MQA
```
Comment (peterjunpark) [inline .wordlist.txt:197]: Multi-Query Attention

File: .wordlist.txt [hunk 19]
```diff
@@ -355,6 +391,7 @@ WikiText
 Wojna
 Workgroups
 Writebacks
+XDL
```
Comment (peterjunpark) [inline .wordlist.txt:394]: Explainable Deep Learning

File: .wordlist.txt [hunk 20]
```diff
@@ -191,6 +206,7 @@ Mellanox
 Mellanox's
 Meta's
 MirroredStrategy
+MoE
```
Comment (peterjunpark) [inline .wordlist.txt:209]: Mixture of Experts

File: .wordlist.txt [hunk 21]
```diff
@@ -108,9 +116,11 @@ GMI
 GPG
 GPR
 GPT
+GPTQ
 GPU
 GPU's
 GPUs
+GQA
```
Comment (peterjunpark) [inline .wordlist.txt:123]: Group-Query Attention

**Review by lpaoletti — APPROVED** [review]

**Review by samjwu — CHANGES_REQUESTED** [review]
Python linting check failed because it expects this list to be in lexicographical order

I use https://alphabetizer.flap.tv/ to do this quickly

**Comment by samjwu** [comment]
Not a big deal, but for this type of update, the commitizen keyword most often used is "ci" for commit messages

e.g.: ci(.wordlist.txt): Update spellcheck wordlist

**Review by samjwu — APPROVED** [review]

**Pull request merged**